roles/network_plugin/calico_defaults/defaults/main.yml

```diff
@@ -101,6 +101,10 @@ calico_iptables_lock_timeout_secs: 10
 # Choose Calico iptables backend: "Legacy", "Auto" or "NFT" (FELIX_IPTABLESBACKEND)
 calico_iptables_backend: "Auto"
 
+# Calico NFTable Mode Support
+# Valid option: Disabled (default), Enabled (when kube_proxy_mode is set "nftable")
+calico_nftable_mode: "{{ 'Enabled' if kube_proxy_mode == 'nftable' else 'Disabled' }}"
```

Comment:
Hi @tico88612, I think we also need to check the calico_version for this option, calico supports this mode since v3.29.x(https://docs.tigera.io/calico/3.29/release-notes/#tech-preview-nftables-data-plane)

Reply:
@cyclinder I added this in `roles/network_plugin/calico/templates/calico-node.yml.j2`

Reply:
As we see these two options(kube_proxy_mode, calico_nftable_mode). If calico_version > v3.29.0 and kube_proxy_mode == nftables, we also use the calico_nftable_mode. Maybe it's a little risky in the production, nftables in calico is just a tech preview now, not GA, Tech preview features may be subject to significant changes before they become GA. see https://docs.tigera.io/calico/3.29/getting-started/kubernetes/nftables.

I think we can use nftables in calico _**only when**_  kube_proxy_mode == 'nftable' and calico_version >= v3.29.0 and calico_nftable_mode == enabled.

Reply:
I thought calico will be crashed when kube_proxy_mode set nftable. But I tested it didn't happen. I changed to 'Disabled'.